app/templates/partials/guidance.html

```diff
@@ -0,0 +1,17 @@
+{% import 'macros/helpers.html' as helpers %}
+
+{% for item in guidance_content.content %}
+  {%- if item.title -%}
+    <h3 class="venus">{{item.title}}</h3>
```

Comment:
I spoke to Hamish about it and the only thing he said was about the mci and having div tags to separate into columns, but maybe worth asking again.